### DISCUSSION TITLE
[@types/three] doesn't support THREE.EdgesHelper

### DISCUSSION THREAD

**Discussion by AnweshGangula** [DefinitelyTyped/DefinitelyTyped#65581] [Issues with a @types package] [answered]
The EdgesHelper class is not included in the type definitions provided by the @types/three. I'm getting the following error when I try to use it

> Property 'EdgesHelper' does not exist on type 'typeof import("c:/Users/AnweshGangula/.../node_modules/@types/three/index")'

same issue is mentioned [here](https://github.com/mrdoob/three.js/issues/10681#issuecomment-287921860).

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment]
I'm having trouble finding `EdgesHelper` in the three.js docs or source code, can you point me in the right direction?

**Comment by AnweshGangula** [comment, marked as answer]
Never mind, 
"THREE.EdgesHelper has been removed. Use THREE.EdgesGeometry instead."

[Example](https://stackoverflow.com/a/41031517/6908282):

```js
scene.add( object );
objects.push( object );

// wireframe
var geo = new THREE.EdgesGeometry( object.geometry );
var mat = new THREE.LineBasicMaterial( { color: 0x000000 } );
var wireframe = new THREE.LineSegments( geo, mat );
object.add( wireframe );
```
